types/node-targz/node-targz-tests.ts

```diff
@@ -0,0 +1,44 @@
+import nodeTargz = require('node-targz');
+
+nodeTargz.compress(
+    {
+        source: './foo/bar',
+        destination: './foo/output.tar.gz',
+        level: 6,
+        memLevel: 6,
+    },
+    err => void 0,
```

Comment:
can you replace operator with no-op function? I bet it's not obvious for readers what it does here (and what `void expression` does here)

Reply:
Of course! - fixed in 6bed78540cc2adbeb12c303f3a2ee2620c5780c7